lib/custom_a11y_rules.js

```diff
@@ -0,0 +1,150 @@
+var customRules = {
+  "rules": [
```

Comment:
FYI, single quotes are preferred over double via our [style guide](https://github.com/edx/edx-platform/wiki/Javascript-standards-for-the-edx-platform), which eventually points to [google's guide](https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings).